Vlv2TbltDevicePkg/PlatformPkgGcc.fdf

```diff
@@ -152,6 +158,20 @@ $(FLASH_REGION_FVMAIN_OFFSET)|$(FLASH_REGION_FVMAIN_SIZE)
 gPlatformModuleTokenSpaceGuid.PcdFlashFvMainBase|gPlatformModuleTokenSpaceGuid.PcdFlashFvMainSize
 FV = FVMAIN_COMPACT
 
+  #
+  # Binary Firmware
+  #
+$(FLASH_REGION_BINARY_FIRMWARE_OFFSET)|$(FLASH_REGION_BINARY_FIRMWARE_SIZE)
+gPlatformModuleTokenSpaceGuid.PcdFlashBinFwBase|gPlatformModuleTokenSpaceGuid.PcdFlashBinFwSize
```

Comment:
@krystian-hebel I can see the file in firmware image, but it is shown as a pad file. Is there any possibility to add some description for this file to find it easier?

Reply:
@miczyg1 not using binary file, I think, they're just blobs. In FV a file can have more than one section, in the `User Interface section` is usually a name for binaries, although I didn't research a specification for this.

Reply:
@krystian-hebel it is just harder for for typical user to find the file in current form. However the file is at desired offset, this is the only information about the file we have to search for it for example in UEFI tool. Would be good if we could have something like a RESERVED type of file.